examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt

```diff
@@ -3,4 +3,5 @@ tqdm
 scikit-learn
 fastdownload
 onnx==1.12.0
-onnxruntime==1.13.1
+onnxruntime-openvino==1.13.1
+openvino-dev~=2022.3.0.dev
```

Comment:
Why not the latest version?

Reply:
I aligned the version according to the other samples

Reply:
Which samples? `setup.py` requires only `openvino-dev`, and other tests too. Only PTQ samples use `~=2022.3.0.dev`, which might be updated, but not in this PR.

Reply:
Ok, If we have plans to update PTQ samples by `openvino-dev` I use this version in ONNX sample.